README.md

```diff
@@ -1,2 +1,40 @@
 # java-globaltracer
-Global ThreadLocal Span for implicit propagation, delegating to another Tracer implementation.
+Global Tracer, forwarding to another Tracer implementation.
+
+This library provides two utility classes `ActiveSpanManager` and `GlobalTracer`:
+
+## ActiveSpanManager
+This class:
+ 1. Provides clients with the `ActiveSpanManger.activeSpan()` method to return the
+    _active span_.
+    If there is no active span, the method never returns a `null` refrence, but
+    a `NoopSpan` instead.
+ 2. Modifications to the _active span_ can be made through the `activate` 
+    and `deactivate` methods.
+ 3. Utility `spanAware()` methods to create 'span aware' Runnable and Callable instances
+    that run with the _active span_ from the scheduling thread.
+
+## GlobalTracer
+This class has the following purpose:
+ 1. The `GlobalTracer.tracer()` factory method returning the singleton _global tracer_.
+    If there is no global tracer, a `NoopTracer` is returned instead.
```

Comment:
should it read "until there's a global tracer assigned, a Noop..."? ex in finagle the tracer is lazy attached to the context.